src/test/java/com/rultor/agents/daemons/EndsDaemonITCase.java

```diff
@@ -114,46 +119,54 @@ public void readsExitCodeCorrectly() throws IOException {
     @Test
     @Ignore
     public void deprecatesDefaultImage() throws IOException {
-        final Talk talk = new Talk.InFile();
-        FileUtils.write(
-            new File(this.start(talk, "").getAbsolutePath(), "testing"), "12"
-        );
-        new EndsDaemon(new Profile.Fixed()).execute(talk);
-        for (final String path
-            : talk.read().xpath("/p/entry[@key='merge']/entry[@key='script']")
+        try (
+            final StartsDockerDaemon start =
+                new StartsDockerDaemon(Profile.EMPTY)
         ) {
-            if ("yegor256/rultor".equals(path)) {
-                final String dir = talk.read()
-                    .xpath("/talk/daemon/dir/text()").get(0);
-                MatcherAssert.assertThat(
-                    dir,
-                    StringContains.containsString(
-                        "#### Deprecation Notice ####"
+            final Talk talk = new Talk.InFile();
+            this.start(start, talk, "");
+            new EndsDaemon(new Profile.Fixed()).execute(talk);
+            for (final String path
+                : talk.read().xpath(
+                "/p/entry[@key='merge']/entry[@key='script']"
                     )
-                );
-                MatcherAssert.assertThat(
-                    dir,
-                    StringEndsWith.endsWith("##############")
-                );
+            ) {
+                if ("yegor256/rultor".equals(path)) {
+                    final String dir = talk.read()
+                        .xpath("/talk/daemon/dir/text()").get(0);
+                    MatcherAssert.assertThat(
+                        dir,
+                        StringContains.containsString(
+                            "#### Deprecation Notice ####"
+                        )
+                    );
+                    MatcherAssert.assertThat(
+                        dir,
+                        StringEndsWith.endsWith("##############")
+                    );
+                }
             }
         }
     }
 
     /**
      * Start a talk.
+     * @param start Docker daemon starter
      * @param talk Talk to start
      * @param stdout Std out
      * @return Home
      * @throws IOException In case of error.
      */
-    private File start(final Talk talk, final String stdout)
-        throws IOException {
-        Assume.assumeFalse(SystemUtils.IS_OS_WINDOWS);
-        final SSHD sshd = new SSHD(this.temp.newFolder());
+    private PfShell start(final StartsDockerDaemon start, final Talk talk,
+        final String stdout) throws IOException {
+        Assume.assumeTrue(
+            "true".equalsIgnoreCase(System.getProperty("run-docker-tests"))
```

Comment:
@mkordas agreed, frankly I dislike even implementing it like this, but we will not ( at least short and probably mid-term be able to run these tests on Rultor, since we cannot give the Rultor container access to a Docker daemon for security reasons :/ Otherwise I'd just not add this assumption.)

But in this concrete case, I think I know of no way to dry this up really. I mean we need that `Assume` part out there, so we're basically asking to dry up the `"run-docker-tests"`, which is just one string. Obviously we could encapsulate that somehow, but all approaches I can think of would rather do a big disservice to readability :/
Any suggestions ? :)